baseplate/diagnostics/tracing.py

```diff
@@ -81,12 +91,26 @@ def __init__(self, service_name,
                 batch_wait_interval=span_batch_interval,
             )
 
+    def force_sampling(self, span):
+        return span.flags and (span.flags & FLAGS['DEBUG'])
+
+    def should_sample(self, span):
+        if span.sampled is None:
+            return (random.uniform(0, 1) < self.sample_rate or
```

Comment:
`uniform(0,1)` gives you a value in `[0,1]` whereas `random.random()` gives you a value in `[0, 1)` ..... which is what I actually want here since I do a `<` check. Fixing!